splunkrepl.js

```diff
@@ -219,7 +225,12 @@ function eval(cmd, context, filename, callback) {
         cmd_cls(cmd, callback);
     }
     else if (cmd == ":exit") {
-        process.exit();
+        if (realtimeState.running) {
```

Comment:
I don't think this is needed anymore as the realtime search exists before you get to the cmd prompt with your last change right?